virt/install/installing-ove.adoc

```diff
@@ -0,0 +1,52 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="installing-ove"]
+= Installing an {ove} cluster
+:context: installing-ove
+
+toc::[]
+
+The following procedures describe a method to install a {ove-first} cluster that uses a simplified user interface, and does not require you to set up an external image registry in disconnected environments, which is a requirement for all other disconnected installation methods.
+
+:FeatureName: Installing an {ove} cluster using this method
+include::snippets/technology-preview.adoc[]
+
+// Is it bad to mix mentions of OVE above and plain Virt below?
+
+This method has the following advantages for users who want to install a cluster primarily to run virtualized workloads using {VirtProductName}:
+
+Simplified web interface:: The cluster installation is performed through a graphical user interface.
+In addition to providing reasonable default configurations, this interface guides you through the installation by providing hints and warnings about configuring your cluster, without limiting the configurations that are available at installation time.
+This reduces the need for in-depth knowledge of {product-title} while still allowing for complex configurations to be made.
+
+No external registry needed:: If you are installing your cluster in a disconnected environment, you do not need to configure an external image registry in your environment when using this method.
+All other disconnected installation methods require this additional environment setup.
+
+Included Operator bundle:: You can install the Virtualization bundle, which includes all of the additional {olm-first} Operators needed to run virtual machines on your cluster, at the same time as the cluster installation.
+
+// Probably need some section for limitations such as bare metal machines, unless that's a commonly understood requirement for virt.
+
+// Downloading the installation ISO
+include::modules/installing-ove-iso.adoc[leveloffset=+1]
+
+// Mounting the ISO and booting the machines
+include::modules/installing-ove-booting.adoc[leveloffset=+1]
+
+// Configuring the cluster installation from the web console
+include::modules/installing-ove-console.adoc[leveloffset=+1]
+
+During the installation process, the rendezvous node reboots and the console you used to configure the installation is no longer accessible.
+At that point, the URL of the cluster web console is provided, however this console is not accessible until the cluster installation is completed.
```

Comment:
I would either make it two sentences  _is provided. However, this_ 

Or change however to _although_ or _but_